core/src/test/java/io/questdb/test/griffin/engine/groupby/SampleByTest.java

```diff
@@ -1327,7 +1327,8 @@ public void testIndexSampleByAlignToCalendarBindVariables() throws Exception {
                         "2021-03-28T02:15:00.000000Z\ta\t103.7167928478985\t128.42101395467057\n";
 
                 String expectedPrague = "k\ts\tlat\tlon\n" +
-                        "2021-03-28T00:10:00.000000Z\ta\t144.77803379943109\tnull\n" +
+                        "2021-03-28T00:10:00.000000Z\ta\t144.77803379943109\t15.276535618609202\n" +
+                        "2021-03-28T01:10:00.000000Z\ta\tnull\tnull\n" +
```

Comment:
Why do we have 2 records with the same timestamp `2021-03-28T01:10:00`? I'd expect them to be aggregated together, even when they are DST edges.

Reply:
That's the side effect of the WITH OFFSET clause. +10 minutes push certain buckets to the gap in the local time, 02:00-03:00 which is mapped by `to_utc()` function to the "next" UTC hour. Added a clarifying comment in 706768a0a40de8c081505432f0b34cf81b5c0083